src/components/redeemPosition/PositionPreview/index.tsx

```diff
@@ -20,9 +20,15 @@ interface Props {
 }
 
 export const PositionPreview = ({ condition, position }: Props) => {
-  const { balanceERC1155 } = useBalanceForPosition(position?.id || '')
+  const { balanceERC1155, refetch } = useBalanceForPosition(position?.id || '')
   const { collateral: token } = useCollateral(position ? position.collateralToken.id : '')
 
+  useEffect(() => {
+    if (position) {
+      refetch()
+    }
+  }, [position, refetch])
+
```

Comment:
This fixes the problem that after redeeming a position and trying to redeem again but this time from the position list you get the old balance on the preview.